scripts/build/Platform/Android/pipeline.json

```
@@ -1,7 +1,7 @@
 {
     "ENV": {
         "GRADLE_HOME": "C:/Gradle/gradle-7.0",
-        "NODE_LABEL": "windows-latest",
+        "NODE_LABEL": "windows-2024-07",
         "LY_3RDPARTY_PATH": "D:\\workspace\\3rdParty",
         "LY_NDK_DIR": "C:/Android/android-sdk/ndk/21.4.7075529",
```

Are we using the older `Android` directory or the new one from `AndroidSdk`? If the latter:
```suggestion
        "LY_NDK_DIR": "C:/AndroidSdk/ndk/21.4.7075529",
```